### PR TITLE
WGLC Feedback

### PR DESCRIPTION
**Pull request by linuxwolf**
Placeholder for resolving WGLC feedback

### CODE REVIEW

**Review by mathiasbynens — APPROVED** [review]

File: javascript-mjs/draft.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ author:
...
-    organization: Mozilla
+    organization:
     email: linuxwolf+ietf@outer-planes.net
  -
     ins: M. Borins
     name: Myles Borins
-    organization: Google
-    email: mylesborins@google.com
+    organization: GitHub
+    email: mylesborins@github.com
```
Comment (mathiasbynens) [inline javascript-mjs/draft.md:26]: @MylesBorins PTAL

Comment (mathiasbynens) [reply javascript-mjs/draft.md:26]: @MylesBorins ping

**Comment by mathiasbynens** [comment]
I’ll go ahead and merge this already, since I’d like to get the `Makefile` changes in master asap. If any feedback comes up we can address it separately.

**Comment by MylesBorins** [comment]
Sorry I slept on reviewing this... will do better.

**Pull request merged**